### DISCUSSION TITLE
Universe 2024 Keynote speaker....is you?!

### DISCUSSION THREAD

**Discussion by novium2** [community/community#142340] [Other Feature Feedback, Questions, & Ideas]
You're at Universe 2024 when suddenly, you find yourself in the spotlight. What talk are you giving for 15 minutes with no prep?

**Comment by novium2** [comment]
Mine would definitely be the importance of companies with open source products sketching out exactly where community contributions fit into their processes in the repo (and outside of it) and how community contributors will be kept in the loop (and also, to teach your developers to be nice to them, haha).

**Reply by slowboii** [reply]
I'd start talking about how facinating fruit growing out of nowhere is kinda fascinating, compare it to pregnancy and ask people to vote for any party that wants to legalize weed and exit the stage by saying how excited im for the speech i'm doing next year.

Yeah, that's about it.

**Reply by slowboii** [reply]
i'm way too satisfied with myself with a bogus answer like that, i have mixed feelings.

**Comment by Mohataseem89** [comment]
If I had to give a 15-minute Universe talk with zero prep, I will probably speak about how accessible building things has gotten due to open source and modern tooling.

from learning the basics to full-stack app deployments, the barrier to entry has dropped massively, and that shift has empowered so many new developers to build real projects and grow through community-driven learning.

**Comment by Winorlooseiwin** [comment]
Being a community contributor i would probably discuss how making integrated access is important for those who may not know exactly what to do.
